pkg/gardenlet/apis/config/v1alpha1/defaults_test.go

```diff
@@ -60,9 +63,40 @@ var _ = Describe("Defaults", func() {
 			Expect(obj.Server.HTTPS.Port).To(Equal(2720))
 			Expect(obj.SNI).ToNot(BeNil())
 			Expect(obj.SNI.Ingress).ToNot(BeNil())
-			Expect(obj.SNI.Ingress.Labels).To(Equal(map[string]string{"istio": "ingressgateway"}))
 			Expect(obj.SNI.Ingress.Namespace).To(PointTo(Equal("istio-ingress")))
 			Expect(obj.SNI.Ingress.ServiceName).To(PointTo(Equal("istio-ingressgateway")))
+			Expect(obj.SNI.Ingress.Labels).To(Equal(map[string]string{
+				v1beta1constants.LabelApp: "istio-ingressgateway",
+				"istio":                   "ingressgateway",
```

Comment:
Why do we still check for the label `"istio": "ingressgateway"` when it is removed in other places?

Reply:
Same as above.